pkg/k8shandler/logstore.go

```diff
@@ -288,9 +288,9 @@ func (clusterRequest *ClusterLoggingRequest) removeElasticsearchCR(elasticsearch
 
 func (clusterRequest *ClusterLoggingRequest) createOrUpdateElasticsearchCR() (err error) {
 
-	if !clusterRequest.isManaged() {
-		return nil
-	}
+	//if !clusterRequest.isManaged() {
```

Comment:
This makes sense, we shouldn't need this guard anymore

Reply:
We shouldn't be leaving this commented out. If we don't need it - remove it, there's revision history in git we can go back to.